### PR TITLE
DATA: Implement Model & Migrations for GroupPost

### LINKED ISSUE #168: DATA: Implement Model & Migrations for GroupPost
```csharp
public class GroupPost
{
    public Guid GroupId {get; set;}
    public Group Group {get; set;}
    
    public Guid PostId {get; set;}
    public Post Post {get; set;}
}
```

Helpful videos to show how to finalize this task:
https://www.youtube.com/watch?v=6sD5zz9BuOk
https://www.youtube.com/watch?v=XoDwJ-dvhI0

https://github.com/hassanhabib/The-Standard/blob/master/0.%20Introduction/0.0%20Purposing%2C%20Modeling%20%26%20Simulation.md

Sample task for the same model:
https://github.com/hassanhabib/Taarafo.Core/pull/161/files

This task is assigned to @deepshri7

### PR DESCRIPTION
**Pull request by rcruzfreelance**
Close #168

### CODE REVIEW

File: Taarafo.Core/Migrations/20220401024818_AddGroupPostModel.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
```
Comment (ShriHumrudha) [inline Taarafo.Core/Migrations/20220401024818_AddGroupPostModel.cs:1]: Add copyrights header

File: Taarafo.Core/Models/Groups/Group.cs [hunk 2]
```diff
@@ -15,5 +18,9 @@ public class Group
         public GroupStatus Status { get; set; }
         public DateTimeOffset CreatedDate { get; set; }
         public DateTimeOffset UpdatedDate { get; set; }
+
+        [JsonIgnore]
+        public IEnumerable<GroupPost> GroupPosts { get; set; }
+
```
Comment (ShriHumrudha) [inline Taarafo.Core/Models/Groups/Group.cs:24]: Remove extra line

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: Taarafo.Core/Migrations/20220401024818_AddGroupPostModel.cs (hunk 1 above)
Comment (rcruzfreelance) [reply Taarafo.Core/Migrations/20220401024818_AddGroupPostModel.cs:1]: done

File: Taarafo.Core/Models/Groups/Group.cs (hunk 2 above)
Comment (rcruzfreelance) [reply Taarafo.Core/Models/Groups/Group.cs:24]: done

**Review by ShriHumrudha — APPROVED** [review]

**Comment by rcruzfreelance** [comment]
Can anybody merge these changes they been already approved by @ShriHumrudha

**Comment by ShriHumrudha** [comment]
@rcruzfreelance Please address the conflicts

**Comment by rcruzfreelance** [comment]
done

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**